### PR TITLE
Tweak link in README

### PR DESCRIPTION
**Pull request by chausner**
~~This removes a paragraph from the README that referred to the manifest feature of vcpkg as experimental. Since manifests are considered a stable feature by now, I suggest to remove this note.~~

The second change turns a path in the README into a clickable link.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,11 +55,6 @@ and potentially add the port to vcpkg.
 After you've gotten vcpkg installed and working,
 you may wish to add [tab completion](#tab-completionauto-completion) to your shell.
 
-Finally, if you're interested in the future of vcpkg,
-check out the [manifest][getting-started:manifest-spec] guide!
```
Comment (autoantwort) [inline README.md:59]: Imho this should be promoted more. Manifest mode is not mentioned in the README but should be the default for cmake projects.

**Comment by BillyONeal** [comment]
I think manifest mode is still experimental for as long as we can't call it our front line / default usage scenario, and too many commands (e.g. search) don't work with it.

Don't get me wrong, I like the feature, and we do need to flesh out that story to make it as nice to use as classic mode is, but I don't think we are there yet.

**Comment by chausner** [comment]
> I think manifest mode is still experimental for as long as we can't call it our front line / default usage scenario, and too many commands (e.g. search) don't work with it.

Thanks, I wasn't aware of these limitations and that you're considering the feature still experimental. I'll close this MR then.

**Comment by chausner** [comment]
In fact, let me reopen the MR instead and revert the changes about the manifest mode. The other tweak to the README is still valid.

**Review by Adela0814 — DISMISSED** [review]

**Comment by BillyONeal** [comment]
I'm dropping reviewed from these because I'm not sure how they are affected by the ongoing transition to learn.microsoft.com. /cc @ras0219-msft @AugP

**Review by JavierMatosD — DISMISSED** [review]

**Review by ras0219-msft — APPROVED** [review]

**Comment by Adela0814** [comment]
The relevant files of this PR do not exist in the current [vcpkh-doc](https://github.com/microsoft/vcpkg-docs), so it cannot be migrated temporarily.

**Review by ras0219-msft — DISMISSED** [review]
Thanks for the PR!

We're in the process of migrating our documentation to https://learn.microsoft.com/vcpkg (repo: https://github.com/Microsoft/vcpkg-docs). Therefore, the privacy links should change to point to https://learn.microsoft.com/vcpkg/about/privacy.

Please convert the link text and linked pages to `https://learn.microsoft.com/vcpkg/about/privacy`.

**Comment by chausner** [comment]
I adapted the link as suggested.

**Review by Adela0814 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Please resolve conflicts

**Comment by chausner** [comment]
> Please resolve conflicts

Done.

**Review by Adela0814 — APPROVED** [review]

**Pull request merged**